pom.xml

```diff
@@ -253,6 +253,36 @@
         <email>eolivelli@apache.org</email>
         <timezone>+1</timezone>
     </developer>
+    <developer>
+      <id>nkalmar</id>
+      <name>Norbert Kalmar</name>
+      <email>nkalmar@apache.org</email>
+      <timezone>+1</timezone>
+    </developer>
+    <developer>
+      <id>enixon</id>
+      <name>Brian Nixon</name>
+      <email>enixon@apache.org</email>
+      <timezone>-8</timezone>
+    </developer>
+    <developer>
+      <id>symat</id>
+      <name>Mate Szalay-Beko</name>
+      <email>symat@apache.org</email>
+      <timezone>+1</timezone>
+    </developer>
+    <developer>
+      <id>ddiederen</id>
+      <name>Damien Diederen</name>
+      <email>ddiederen@apache.org</email>
+      <timezone>+2</timezone>
```

Comment:
Ah, time zones.  Never been a sticky topic in the history of computers (or the history of history), ever :)

I'm afraid I'm +1 (currently living in Munich).  But that brought the question of DST (Daylight Saving Time), even if it's inactive right now.

Looking at https://maven.apache.org/pom.html#developers, I see:

> **timezone**: A valid time zone ID like `America/New_York` or `Europe/Berlin`, or a numerical offset in hours (and fraction) from UTC where the developer lives, e.g., `-5` or `+1`. Time zone IDs are highly preferred because they are not affected by DST and time zone shifts. […]

So: `<timezone>Europe/Berlin</timezone>`?

Reply:
Got it. Haha, I like this rigorous expression:)

Reply:
:)